language/move-vm/runtime/src/native_functions.rs

```diff
@@ -118,6 +120,13 @@ impl<'a, 'b> NativeContext<'a, 'b> {
             .debug_print_stack_trace(buf, self.resolver.loader())
     }
 
+    pub fn exists_at(&mut self, address: AccountAddress, type_: &Type) -> VMResult<bool> {
```

Comment:
Thats not enough because of the gas problem. (A TODO in my PR.) It can be rather complex to get this right, because native function gas is not designed for dealing with storage.

Also, I think it is not a good practice to introduce those magic native logic and use it somewhere underneath the object model without at the same time making it generally available. We do this already too often (SignerCap). My feeling for clean system design shudders, but my concerns are theoretic, and it already happens anyway. Approving this to unblock you.